src/components/ConcludeStep.tsx

```diff
@@ -13,21 +13,16 @@ export default class ConcludeStep extends React.PureComponent<IProps> {
     const { handleReturnToOpponentSelection, winnings } = this.props;
 
     return (
-      <div className={css(styles.container)}>
+      <React.Fragment>
```

Comment:
@tomclose : Yup! React.Fragment is used when you want to render the children directly, rather than rendering a <div> with the children inside. This is a good idea in this case because we are already wrapping each of the Step components with a container wrapper div in `GameController`. As an example, we have to choose between:
a. wrapping in a div, which renders markup like so:
```
<div> <-- container div from GameController
  <div> <-- div from ConcludeStep
    ... rest of component
  </div>
</div>
```
or, with React.Fragment just rendering its children directly into the parent:
```
<div> <-- container div from GameController
  ... rest of component
</div>
```

Since the inner div is useless, we prefer to render the children directly as to not deepen the DOM tree just because of React's component structuring.